boom! testing postgres_large_object-0.5.1 against nightly for nightly
boom! running: cargo +nightly build --frozen
boom! creating container for: cargo +nightly build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly build --frozen cargobomb`
blam! d46b9c045a9d8f7eeea97de5c87e912d75b1b3999008a7223f922fd96c35b95f
boom! running `docker start -a d46b9c045a9d8f7eeea97de5c87e912d75b1b3999008a7223f922fd96c35b95f`
kablam!    Compiling postgres_large_object v0.5.1 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 2.41 secs
boom! running `docker rm -f d46b9c045a9d8f7eeea97de5c87e912d75b1b3999008a7223f922fd96c35b95f`
blam! d46b9c045a9d8f7eeea97de5c87e912d75b1b3999008a7223f922fd96c35b95f
boom! running: cargo +nightly test --frozen --no-run
boom! creating container for: cargo +nightly test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen --no-run cargobomb`
blam! 6316699fa70ebc69d2aac9e10b26691304c3defe24e2194640d62b8b05571725
boom! running `docker start -a 6316699fa70ebc69d2aac9e10b26691304c3defe24e2194640d62b8b05571725`
kablam!    Compiling postgres_large_object v0.5.1 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 5.4 secs
boom! running `docker rm -f 6316699fa70ebc69d2aac9e10b26691304c3defe24e2194640d62b8b05571725`
blam! 6316699fa70ebc69d2aac9e10b26691304c3defe24e2194640d62b8b05571725
boom! running: cargo +nightly test --frozen
boom! creating container for: cargo +nightly test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen cargobomb`
blam! 32a1a259a718c152a7dd1a45af4cdf4254d5b220550e34553c45a44c45ce107a
boom! running `docker start -a 32a1a259a718c152a7dd1a45af4cdf4254d5b220550e34553c45a44c45ce107a`
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 0.1 secs
kablam!      Running /target/debug/deps/postgres_large_object-570299ae5e12bafd
blam! 
blam! running 8 tests
blam! test test::test_delete_bogus ... FAILED
blam! test test::test_create_delete ... FAILED
blam! test test::test_open_finish ... FAILED
blam! test test::test_open_bogus ... FAILED
blam! test test::test_seek_tell ... FAILED
blam! test test::test_truncate ... FAILED
blam! test test::test_write_with_read_fd ... FAILED
blam! test test::test_write_read ... FAILED
blam! 
blam! failures:
blam! 
blam! ---- test::test_delete_bogus stdout ----
blam! 	thread 'test::test_delete_bogus' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /checkout/src/libcore/result.rs:859
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- test::test_create_delete stdout ----
blam! 	thread 'test::test_create_delete' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- test::test_open_finish stdout ----
blam! 	thread 'test::test_open_finish' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- test::test_open_bogus stdout ----
blam! 	thread 'test::test_open_bogus' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- test::test_seek_tell stdout ----
blam! 	thread 'test::test_seek_tell' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- test::test_truncate stdout ----
blam! 	thread 'test::test_truncate' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- test::test_write_with_read_fd stdout ----
blam! 	thread 'test::test_write_with_read_fd' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- test::test_write_read stdout ----
blam! 	thread 'test::test_write_read' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /checkout/src/libcore/result.rs:859
blam! 
blam! 
blam! failures:
blam!     test::test_create_delete
blam!     test::test_delete_bogus
blam!     test::test_open_bogus
blam!     test::test_open_finish
blam!     test::test_seek_tell
blam!     test::test_truncate
blam!     test::test_write_read
blam!     test::test_write_with_read_fd
blam! 
blam! test result: FAILED. 0 passed; 8 failed; 0 ignored; 0 measured
blam! 
kablam! error: test failed
boom! running `docker rm -f 32a1a259a718c152a7dd1a45af4cdf4254d5b220550e34553c45a44c45ce107a`
blam! 32a1a259a718c152a7dd1a45af4cdf4254d5b220550e34553c45a44c45ce107a
